ci-operator/infra/openshift/release-controller/releases/release-ocp-4.2.json

```diff
@@ -78,6 +78,10 @@
       "optional":true,
       "prowJob":{"name":"canary-openshift-ocp-installer-e2e-openstack-4.2"}
     },
+    "openstack":{
```

Comment:
This should be changed to `openstack-serial`

Reply:
Good catch. I just pushed an update to fix this.